server/routerlicious/packages/services-core/src/clusterDraining.ts

```diff
@@ -14,3 +14,9 @@ export interface IClusterDrainingChecker {
 	 */
 	isClusterDraining(cluster?: string): Promise<boolean>;
 }
+
+/**
+ * Retry after time when cluster is under draining
+ * @internal
+ */
+export const clusterDrainingRetryTimeInMs = (10 + 2) * 60 * 1000; // 12 minutes
```

Comment:
just curious how did we come up with 12minutes?

Reply:
10 minutes is the session grace period. partitionActivityTimeout in server/routerlicious/packages/services-core/src/configuration.ts. Deli lambda will be closed after this timeout and during the lambda closure, it will create global checkpoint and scribe will create service summary. 12 minutes is the 10 minutes session grace period plus a 2 minutes buffer.